apps/frontend/src/app/5_pages/StakePage/components/VestingStakesFrame/VestingStakesFrame.types.ts

```diff
@@ -0,0 +1,21 @@
+export type VestingData = {
+  vestingAddress: string;
+  vestingCreationType: string;
+  vestingType: string;
+};
+
+export type VestingGroup =
```

Comment:
This will be adjusted once Staking and Vesting PRs are in the same branch because we already have it implemented in https://github.com/DistributedCollective/sovryn-dapp/pull/436.